app/components/contest/index.js

```diff
@@ -0,0 +1,170 @@
+    <div className="central-container">
+      <h2>
+        {' '}
+        <span className="logojs">JSHeroes</span> Sticker Contest
+      </h2>
+      <section>
+        <strong>The creative brief</strong>
+        <div>
+          JSHeroes is an <strong>open-source</strong> community event, this means we rely on the
+          free time and skills of the community to organize the event each year. We are currently
+          preparing the 3rd edition taking place in April 11-12, 2019.
+        </div>
+        <div>
+          We love everything related to communities, and
+          <strong> we love stickers</strong> - we collect them, trade them, and somethime even stick
+          them on our laptops. Some of us like to use them on mugs and T-shirts, whatever the case,
+          we love these things!
+        </div>
+        <div>
+          Our aim is to expand our pool of stickers with new and creative ones that people would
+          love to use. The people we reach are web/software developers, UI/UX designers.
+        </div>
+      </section>
+
+      <section>
+        <strong>Prizes</strong>
+        <div>
+          <strong>1st</strong> PRIZE: 1 ticket to JSHeroes conference
+        </div>
+        <div>
+          <strong>2nd</strong> PRIZE: 1 ticket to JSHeroes conference
+        </div>
+        <div>
+          <strong>3rd</strong> PRIZE: Design and UI/UX books you’ll love
+        </div>
+        <div>
+          Each sticker will be voted on individually and all submissions will be ranked according to
+          the number of votes. If a candidate has more than 1 sticker selected, only the top 1st
+          sticker will receive a prize.
+        </div>
+      </section>
+
+      <section>
+        <strong>Competition schedule</strong>
+        <div>
+          Sticker submission: <strong>January 16 – 26</strong>, 2019
+        </div>
+        <div>
+          JSHeroes team voting: <strong>January 27-28</strong>, 2019
+        </div>
+        <div>
+          Winners announcement: <strong>January 31</strong>, 2019
+        </div>
+      </section>
+
+      <section>
+        <strong>Requirements</strong>
+        <div>• Each candidate can submit a maximum of two stickers.</div>
+        <div>
+          • All images must be submitted as a high-resolution, Adobe Illustrator or Photoshop file
+          AND a JPEG at 300 dpi or higher.
+        </div>
+      </section>
+
+      <section>
+        <strong>Design criteria</strong>
+        <div>
+          1. Stickers designs must be original work and incorporate the{' '}
+          <strong>JSHeroes logo</strong>
+        </div>
+        <div>2. JavaScript technology / community-related</div>
+        <div>3. Be simple, be funny if possible.</div>
+      </section>
+
+      <section>
+        <strong>Submission rules</strong>
+        <div>
+          Please submit your stickers as a compressed or zip file to: <u>welcome@jsheroes.io</u>
```

Comment:
I do not think it is necessary.
This part has more the role of informing the user that he can use this email adress after he has finished his artwork. What if he does not have an email manager installed? :)) 

I don't know actually! It could also be usefull.

Reply:
Agree that it's not a must-have.

Email clients/managers come with every OS (for e.g. Outlook on Windows and Mail app on mac). For people that use them it could be a nice experience to click on the link and have the email client open with the subject pre-filled.